projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -47,14 +58,18 @@ export class CartEffects {
   );
 
   @Effect()
-  createCart$: Observable<any> = this.actions$.pipe(
+  createCart$: Observable<
+    | fromActions.MergeCartSuccess
```

Comment:
remove this pipe at the begging of the type

Reply:
its prettier config